### PR TITLE
Issue/1253

### PR DESCRIPTION
**Pull request by mehul0810**
## Description
This PR is for #1253 

## How Has This Been Tested?
In Progress

## Types of changes
Bug fix (non-breaking change which fixes an issue)

## Checklist:
- [x] My code is tested.
- [ ] My code follows the WordPress code style.
- [ ] My code follows has proper inline documentation.

### CODE REVIEW

**Comment by DevinWalker** [comment]
@mehul0810 I just ran checked it out, ran the upgrade, confirmed the meta keys for the min/max level amounts were created per form. Then I went to check the sorting and it doesn't appear they are ordering properly:



Am I missing something?

**Comment by mehul0810** [comment]
@DevinWalker Yes, It is still in progress as not getting expected results using WP_Query with `meta_query`. Created this PR so that everyone can give their inputs and @ravinderk can see my commits as per our discussion over call yesterday.

**Comment by DevinWalker** [comment]
Gotcha, I just wanted to make sure. I'll take a look and see if I can figure out a solution.

File: includes/admin/forms/class-metabox-form-data.php [hunk 1]
```diff
@@ -836,18 +836,29 @@ public function save( $post_id, $post ) {
...
-								give_update_meta( $post_id, $form_meta_key, $form_meta_value );
 						}
 
+
+						/**
+						 * Filter the form meta value before saving
+						 *
+						 * @since 1.8.9
+						 */
+						$form_meta_value = apply_filters(
```
Comment (ravinderk) [inline includes/admin/forms/class-metabox-form-data.php]: @DevinWalker I saw some form meta value:
1.`_give_set_value`
2. `_give_custom_amount_minimum`
3. `_give_donation_levels[$index][_give_amount]`

storing with currency formatting which is not good. Amount should store only with decimal formatting.

File: includes/admin/upgrades/upgrade-functions.php [hunk 2]
```diff
@@ -909,19 +918,125 @@ function give_v188_upgrades() {
...
+		while ( $donation_forms->have_posts() ) {
+			$donation_forms->the_post();
+			$form_id = get_the_ID();
 
-    foreach ( $caps_to_add as $cap ) {
-        // Add the capability.
-        $give_worker->add_cap( $cap );
-    }
+			// Remove formatting from _give_set_price
+			update_post_meta(
```
Comment (ravinderk) [inline includes/admin/upgrades/upgrade-functions.php]: @DevinWalker I added upgrade script to sanitize amount values:
1. `_give_set_value`
2. `_give_custom_amount_minimum`
3. `_give_donation_levels[$index][_give_amount]`

**Comment by ravinderk** [comment]
@DevinWalker This PR is ready to test.

**Pull request merged**